.circleci/regenerate.py

```diff
@@ -268,9 +268,9 @@ def cmake_workflows(indentation=6):
         for device in device_types:
             job = {"name": f"cmake_{os_type}_{device}", "python_version": python_version}
 
-            job["cu_version"] = "cu102" if device == "gpu" else "cpu"
+            job["cu_version"] = "cu111" if device == "gpu" else "cpu"
```

Comment:
Do we have to align with PyTorch and use CUDA 11.3 for which we get nightly binaries OR it's OK to stick with 11.1? Are there any incompatibilities we should be aware of?

Reply:
Yes, moving it to 11.3, don't want to mess us with 11.1 here